EnglishFilter/sections/general_extensions.txt

```diff
@@ -257,6 +257,9 @@ streamsport.icu#%#//scriptlet('prevent-window-open')
 ontiva.com#%#//scriptlet("prevent-window-open")
 ! https://github.com/AdguardTeam/AdguardFilters/issues/71112
 adbl.live#%#//scriptlet("prevent-window-open")
+! https://github.com/AdguardTeam/AdguardFilters/issues/71492
+fullpornfuck.com#%#//scriptlet('prevent-window-open', '1', 'getalltraffic')
+fullpornfuck.com$$script[tag-content="popit"][max-length="1000"]
```

Comment:
`! 'popit' (in general it is popular on fake NSFW sites, and prevets to close a tab)`

It's better to use the rules under this comment